subsys/bluetooth/mesh/lpn.c

```diff
@@ -911,8 +916,7 @@ static void lpn_timeout(struct k_work *work)
 		clear_friendship(true, false);
 		break;
 	case BT_MESH_LPN_RECV_DELAY:
-		k_work_reschedule(&lpn->timer,
-				  K_MSEC(lpn->adv_duration + SCAN_LATENCY + lpn->recv_win));
+		k_work_reschedule(&lpn->timer, K_MSEC(SCAN_LATENCY + lpn->recv_win));
```

Comment:
If Receive Delay was started `RX_DELAY_CORRECTION` ms earlier, shouldn't Receive Window be extended by this time (like with `SCAN_LATENCY`)?

Reply:
sounds reasonable